content/src/content/jcr_root/apps/core/wcm/components/container/v1/container/clientlibs/design/js/container.js

```diff
@@ -0,0 +1,60 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2019 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+/* global jQuery */
+(function($) {
+    "use strict";
+
+    var dialogContentSelector = ".cmp-container__design";
+    var colorHiddenCheckboxSelector = 'coral-checkbox[name="./colorsDisabled"]';
+    var colorPropertyHiddenCheckboxSelector = 'coral-checkbox[name="./propertyDisabled"]';
+    var colorMultifieldSelector = ".cmp-container__design-multifield";
+    var swatchesListLabelSelector = ".coral-Form-fieldlabel-swatcheslist";

```

Comment:
Please define a `selectors` object, similar to:
https://github.com/adobe/aem-core-wcm-components/blob/feature/container/content/src/content/jcr_root/apps/core/wcm/components/carousel/v1/carousel/clientlibs/editor/js/carousel.js#L20